pipeline/run_beam_tables.py

```diff
@@ -54,13 +57,17 @@ def run_parallel_pipelines(runner: beam_tables.ScanDataBeamPipelineRunner,
   with concurrent.futures.ThreadPoolExecutor() as pool:
     futures = []
     for scan_type in scan_types:
-      table_name = beam_tables.get_table_name(dataset, scan_type,
-                                              beam_tables.BASE_TABLE_NAME)
-      job_name = beam_tables.get_job_name(table_name, incremental_load)
+      destination = beam_tables.get_table_name(dataset, scan_type,
+                                               beam_tables.BASE_TABLE_NAME)
+      if export_gcs:
+        # TODO: define global for bucket
+        destination = beam_tables.get_gcs_folder(dataset, scan_type,
+                                                 'firehook-test')
```

Comment:
We can keep this value for now, but this should be defined in `firehook_resources.py`

Reply:
Also, not a change for this PR. But eventually we're going to want both a prod and dev bucket so we're not writing dev stuff to a prod bucket we probably want to make world-readable.

Reply:
Moved to `firehook_resources.py` and added a TODO for separate prod and dev buckets